### PR TITLE
User login now creates empty offers table for new users

### PR DESCRIPTION
**Pull request by Shalev-Lifshitz**

### CODE REVIEW

File: src/main/java/tech/autodirect/api/database/TableUsers.java [hunk 1]
```diff
@@ -43,22 +45,25 @@ public void addUser(
...
-        stmt.setString(5, TableOffersInterface.createTableName(userId));
-
-        // execute and close the above SQL statement
-        stmt.executeUpdate();
-        stmt.close();
-
-        // TODO: What happens if user already exists?
-        // TODO: What happens if there is no offers table for this user yet (which is usually the case)?
+    ) throws SQLException, ClassNotFoundException {
+        if (!userExists(userId)) {
```
Comment (sammdu) [inline src/main/java/tech/autodirect/api/database/TableUsers.java:49]: Why are we still doing this?
It should throw an error

Comment (sammdu) [reply src/main/java/tech/autodirect/api/database/TableUsers.java:49]: Let's explicitly do the check in SvcUserLogin, and create a table there explicitly.

**Review by tupi2001 — DISMISSED** [review]
Looks good!

**Comment by sammdu** [comment]
> Looks good!

No no there's an issue here still.

**Review by tupi2001 — APPROVED** [review]
Once again. This has been reviewed and I approve

**Pull request merged**